cypress/apiRequests/Attribute.js

```diff
@@ -58,3 +58,21 @@ export function deleteAttribute(attributeId) {
   }`;
   return cy.sendRequestWithQuery(mutation);
 }
+
+export function getAttribute(attributeId) {
+  const query = `query{
```

Comment:
I'm wondering if we can use `graphql-tag` to validate queries or at least get autocomplete + formatting